home/home2.1/src/Main.java

```diff
@@ -0,0 +1,92 @@
+import java.io.*;
+import java.util.*;
+
+class StatisticsDatabase {
+    private Map<String, Integer> stats;
+    private String filePath;
+
+    public StatisticsDatabase(String filePath) {
+        this.filePath = filePath;
+        this.stats = new HashMap<>();
+    }
+
+    public void load() {
+        try (Scanner scanner = new Scanner(new File(filePath))) {
+            while (scanner.hasNext()) {
+                String name = scanner.next();
+                int count = scanner.nextInt();
+                stats.put(name, count);
+            }
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void save() {
+        try (PrintWriter writer = new PrintWriter(filePath)) {
+            for (Map.Entry<String, Integer> entry : stats.entrySet()) {
+                writer.println(entry.getKey() + " " + entry.getValue());
+            }
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void clear() {
+        stats.clear();
+        saveToFile();
+        System.out.println("The history was deleted");
+    }
+
+    public void deleteUser(String name) {
+        stats.remove(name);
+        saveToFile();
+        System.out.println("Statistics was deleted for " + name);
+    }
+
+    public int incrementUserAndGetCount(String name) {
+        int count = stats.getOrDefault(name, 0);
+        count++;
+        stats.put(name, count);
+        saveToFile();
+        return count;
+    }
+
+    private void saveToFile() {
+        try (PrintWriter writer = new PrintWriter(new FileWriter(filePath))) {
+            for (Map.Entry<String, Integer> entry : stats.entrySet()) {
+                writer.println(entry.getKey() + " " + entry.getValue());
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+}
+
+public class Main {
+    public static void main(String[] args) {
+        if (args.length != 1 && args.length != 2) {
+            System.err.println("Usage: java Main <name> [delete]");
+            System.exit(1);
+        }
+
+        String name = args[0];
+        String databasePath = "statistics.txt";
```

Comment:
it is better to have named constants for filename and delete/bread commands. Named constants allow to make support of the code simpler as you will have a single place that should be changed in case of change request.